### PR TITLE
Update Features - Remove use of get_common_setting

### PR DESCRIPTION
**Pull request by samruddhikhandale**
Closes https://github.com/devcontainers/features/issues/572

As per https://github.com/microsoft/vscode-dev-containers/issues/1762, the repo will be archived soon. Also, there is no reason to depend on a remotely hosted file for fetching common settings.

### CODE REVIEW

**Review by joshspicer — APPROVED** [review]

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-    "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:0-18",
-    "remoteUser": "node",
-    "customizations": {
-        "vscode": {
-            "extensions": [
-                "mads-hartmann.bash-ide-vscode"
-            ]
-        }
-    },
+    "image": "mcr.microsoft.com/devcontainers/javascript-node:1-18",
```
Comment (Adawntoremember) [inline .devcontainer/devcontainer.json:2]: .devcontainer/devcontainer.json

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
         "ghcr.io/devcontainers/features/docker-in-docker:2": {}
     },
     "postCreateCommand": "npm install -g @devcontainers/cli",
     "hostRequirements": {
         "cpus": 4
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "mads-hartmann.bash-ide-vscode"
```
Comment (Adawntoremember) [inline .devcontainer/devcontainer.json:13]:
```suggestion
                "mads-hartmann.bash-ide-vscode"
```

**Review by Adawntoremember — APPROVED** [review]

File: .devcontainer/devcontainer.json [hunk 3]
```diff
@@ -1,18 +1,17 @@
 {
-    "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:0-18",
```
Comment (Ekene1998) [inline .devcontainer/devcontainer.json:2]: "image": "mcr.microsoft.com/devcontainers/javascript-node:0-18",

File: .devcontainer/devcontainer.json
Comment (Ekene1998) [inline .devcontainer/devcontainer.json:1]: "image": "mcr.microsoft.com/devcontainers/javascript-node:0-18",

**Pull request merged**